web/src/features/fireBehaviourCalculator/components/FBCInputGrid.tsx

```diff
@@ -74,106 +121,268 @@ const FBCInputGrid = (props: FBCInputGridProps) => {
     grassCure: row.grassCure,
     windSpeed: row.windSpeed
   }))
 
-  // eslint-disable-next-line
-  const columns: any = React.useMemo(
-    () => [
-      {
-        Header: 'Select',
-        accessor: 'select'
-      },
-      {
-        Header: 'Station',
-        accessor: 'weatherStation'
-      },
-      {
-        Header: 'Fuel Type',
-        accessor: 'fuelType'
-      },
-      {
-        Header: 'Grass Cure %',
-        accessor: 'grassCure'
-      },
-      {
-        Header: 'Wind Speed (km/hr) (Optional)',
-        accessor: 'windSpeed'
-      }
-    ],
-    []
-  )
+  type FBCTableRow = DisplayableInputRow & Partial<FBCStation>
 
-  const tableInstance = useTable({ columns, data })
-
-  const { getTableProps, headerGroups, rows, prepareRow } = tableInstance
-
-  // eslint-disable-next-line
-  const renderHeader = (column: any) => {
-    if (column.id === 'windSpeed') {
-      return (
-        <span>
-          {'Wind Speed (km/h) (Optional)'}
-          <Tooltip title="Leave this empty to calculate forecasted/observed wind speed. Add a custom wind speed to influence the calculations">
-            <InfoIcon aria-label="info"></InfoIcon>
-          </Tooltip>
-        </span>
-      )
-    }
-    if (column.id === 'select') {
-      return (
-        <Checkbox
-          color="primary"
-          checked={headerSelected}
-          onClick={() => {
-            if (headerSelected) {
-              // Toggle off
-              props.updateSelected([])
-              setHeaderSelect(false)
-            } else {
-              props.updateSelected(rows.map(row => parseInt(row.id)))
-              setHeaderSelect(true)
-            }
-          }}
-        />
-      )
+  const rows: FBCTableRow[] = zipWith(
+    inputFieldData,
+    calculatedResults,
+    (inputRow, outputRow) => {
+      if (inputRow) {
+        return [
+          {
+            ...inputRow,
+            ...outputRow
+          }
+        ]
+      }
+      return []
     }
-    return column.render('Header')
-  }
+  ).flat()
+  const DECIMAL_PLACES = 1
+
   return (
-    <Table {...getTableProps()}>
-      <TableHead>
-        {headerGroups.map((headerGroup, hi) => (
-          <TableRow {...headerGroup.getHeaderGroupProps()} key={hi}>
-            {headerGroup.headers.map((column, hci) => (
-              <TableCell {...column.getHeaderProps()} key={hci}>
-                {renderHeader(column)}
-              </TableCell>
-            ))}
-          </TableRow>
-        ))}
-      </TableHead>
-      <TableBody>
-        {rows.map((row, ri) => {
-          prepareRow(row)
-          return (
-            <TableRow {...row.getRowProps()} key={ri}>
-              {row.cells.map((cell, ci) => {
+    <div className={classes.display} data-testid={props.testId}>
+      <Paper className={classes.paper} elevation={1}>
+        <TableContainer className={classes.tableContainer}>
+          <Table stickyHeader aria-label="daily table view of HFI by planning area">
```

Comment:
this aria-label is incorrect (we don't use planning areas), but since we generally have been ignoring the use of aria, I'm not sure how important this is...

Reply:
I can update it to "Fire Behaviour Analysis table"